data.json

```diff
@@ -288,11 +302,15 @@
   },
   {
     "title": "Kill Bill: Volume 1",
-    "emojiImgs": "🔪🙍‍🥋🦶🇯🇵",
+    "emojiImgs": "🗡️🙍‍🥋🦶🇯🇵",
     "genres": ["action", "drama"],
     "type": "movie",
     "year": 2003
   },
+  {
+    "title": "Kill Bill: Volume 2",
+    "emojiImgs": "👰🗡️👧💓💥"
+  },
```

Comment:
Hi! Everything else looks great but looks like Kill Bill Volume 2 is missing the genres, type and year. Could you update your code with that information? Thanks!

Reply:
Ah yes, I'm sorry, fixed now 😁.